### PR TITLE
Remove cruft from modules

### PR DESCRIPTION
**Pull request by dgw**
What started as a fixup in #1400 (noticing that a regex string literal I corrected to pass new style checks was actually unused) turned into a longer-term project to go through all the modules and remove old, orphaned cruft that isn't needed any more.

The `version` module is the first to receive this attention, and I'll check the rest after Hacktoberfest ends.

### CODE REVIEW

**Comment by dgw** [comment]
Pushed back to 7.0.0, so I can focus on the last remaining "more important" fixes for 6.6.0.

**Comment by dgw** [comment]
This is now ready for eyes. There are a lot of little changes, so take your time!

File: sopel/modules/safety.py [hunk 1]
```diff
@@ -84,7 +88,7 @@ def setup(bot):
     loc = os.path.join(bot.config.homedir, 'malwaredomains.txt')
     if os.path.isfile(loc):
         if os.path.getmtime(loc) < time.time() - 24 * 60 * 60 * 7:
-            # File exists but older than one week, update
+            # File exists but older than one week — update it
```
[the remark this answers is not in the document]
Comment (dgw) [reply sopel/modules/safety.py]: Without spaces, generally, but in monospace like this it makes sense to include the spaces.

File: sopel/modules/safety.py [hunk 2]
```diff
@@ -103,7 +107,7 @@ def _download_malwaredomains_db(path):
 @sopel.module.rule(r'(?u).*(https?://\S+).*')
 @sopel.module.priority('high')
 def url_handler(bot, trigger):
-    """ Check for malicious URLs """
+    """Checks for malicious URLs"""
```
[the remark this answers is not in the document]
Comment (dgw) [reply sopel/modules/safety.py]: Hmm… I tried to be consistent and remove periods from most one-line docstrings like this. Multi-line ones are a different case, and maybe we'll further standardize these things in the future. I'm guessing your PR approval means this is good for a first cleanup pass! :wink:

File: sopel/modules/safety.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Exirel) [reply sopel/modules/safety.py]: Oh! Ok, makes sense to me.

File: sopel/modules/safety.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Exirel) [reply sopel/modules/safety.py]: Yup it's perfect. :)

**Comment by Exirel** [comment]
@dgw just :ship: it :wink:

**Pull request merged**